kotlin/settings.gradle.kts

```diff
@@ -16,9 +16,6 @@ dependencyResolutionManagement {
     repositories {
         google()
         mavenCentral()
-        maven {
```

Comment:
don't remove this as we might add snapshot in future

Reply:
Done